vue/components/ui/molecules/table/table.vue

```diff
@@ -1,7 +1,27 @@
 <template>
-    <table class="table" v-bind:class="classes" v-bind:style="style">
+    <table
+        class="table"
+        v-bind:class="{ classes, disableSelection: lastIndexWithShift !== null }"
+        v-bind:style="style"
+    >
+        <global-events
+            v-on:keydown.meta.65.exact.prevent="onCtrlA()"
+            v-on:keydown.ctrl.65.exact.prevent="onCtrlA()"

```

Comment:
when doing CTRL + click above the checkbox, it doesn't work (the checkbox doesn't check)